eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Rules/TestChain.hs

```diff
@@ -253,13 +253,13 @@ ledgerTraceBase chainSt block =
   , txs
   )
   where
-    (Block (BHeader bhb _) txSeq) = block
+    (Block (BHeader bhb _) blockBody) = block
     slot = bheaderSlotNo bhb
     tickedChainSt = tickChainState slot chainSt
     nes = (nesEs . chainNes) tickedChainSt
     pp_ = nes ^. curPParamsEpochStateL
     -- Oldest to Newest first
-    txs = (reverse . toList . fromTxSeq) txSeq -- HERE WE USE SOME SegWit function
+    txs = blockBody ^. txSeqBlockBodyL . to toList . to reverse -- HERE WE USE SOME SegWit function
```

Comment:
I would like to suggest avoiding using `to` like this in general, unless you need to chain more lenses afterwards, eg `txBody ^. inputsTxBodyL . to toList . to head . coinTxOutL`. In other words there is no need to lensify everything.
```suggestion
    txs = reverse $ toList $ blockBody ^. txSeqBlockBodyL -- HERE WE USE SOME SegWit function
```